docs/docs/reference/starlark-instructions.md

```diff
@@ -333,39 +333,22 @@ The previous `GetHttpRequestRecipe` type `GetHttpRequestRecipe(service_name = "m
 :::
 
 ```python
-get_request_recipe = GetHttpRequestRecipe(
-    # The port ID that is the server port for the request
-    # MANDATORY
-    port_id = "my_port",
-
-    # The endpoint for the request
-    # MANDATORY
-    endpoint = "/endpoint?input=data",
-
-    # The extract dictionary takes in key-value pairs where:
-    # Key is a way you refer to the extraction later on
-    # Value is a 'jq' string that contains logic to extract from response body
-    # OPTIONAL
-    extract = {
-        "extracted-field": ".name.id",
-    },
-)
-get_response = plan.request(
+http_response = plan.request(
     # The recipe that will determine the request to be performed.
     # Valid values are of the following types: (GetHttpRequestRecipe, PostHttpRequestRecipe)
     # MANDATORY
-    recipe = get_request_recipe,
+    recipe = request_recipe,
     
     # If the recipe returns a code that does not belong on this list, this instruction will fail.
     # OPTIONAL (Defaults to [200, 201, ...])
     acceptable_codes = [200, 500], # Here both 200 and 500 are valid codes that we want to accept and not fail the instruction
     
-    # If False, instruction will never fail based on code (acceptable_codes will be ignored).
+    # If false, instruction will never fail based on code (acceptable_codes will be ignored).
     # You can chain this call with assert to check codes after request is done.
-    # OPTIONAL (Defaults to False)
-    skip_code_check = False,
+    # OPTIONAL (defaults to false)
+    skip_code_check = false,
```

Comment:
This shouldn't change; the false value in Starlark and Python are indeed  `False` with uppercase

Reply:
```suggestion
    skip_code_check = False,
```